Graylog2/graylog2-server
# Timerange Info for dashboard widgets

## Description
Display the configured time range of a widget in dashboards.

## Motivation and Context
Having a different time range for each widget configured on a dashboard, can make it hard to remember those settings.
So we now display that information in the footer of the widget.

## How Has This Been Tested?
- Create a widget
- Set relative/absolute/keyword timerange
- Check the result in the dashboard

## Types of changes

![Graylog (2)](https://user-images.githubusercontent.com/448763/116102505-0ea34200-a6af-11eb-9eb3-13cdf717a23c.png)
